src/main/java/com/redhat/labs/lodestar/exception/mapper/GitLabServiceResponseMapper.java

```diff
@@ -0,0 +1,32 @@
+package com.redhat.labs.lodestar.exception.mapper;
```

Comment:
purpose for this class?

Reply:
This class make sure the message that comes back from the gitlab API (i.e.  description text of error for things like a 400) is mapped back to the WebApplicationException thrown by the REST client.  Otherwise, that message gets lost and debugging is much harder.

Reply:
👍